packages/stacks-classic/lib/components/pagination/pagination.less

```diff
@@ -9,28 +9,96 @@
 
     & &--item {
         --_pa-item-bg: transparent;
-        --_pa-item-bc: var(--bc-darker);
-        --_pa-item-fc: var(--fc-medium);
-        --_pa-item-bg-focus: var(--black-400);
-        --_pa-item-fc-focus: var(--white);
-        --_pa-item-bg-hover: var(--black-225);
+        --_pa-item-fc: #6B6D73;
+        --_pa-item-bg-hover: #F0EFED;
         --_pa-item-bc-hover: var(--bc-darker);
         --_pa-item-fc-hover: var(--fc-dark);
+        --_pa-item-bb: var(--su-static0); 
 
         // CONTEXTUAL STYLES
-        .highcontrast-mode({ text-decoration: none; });
+        .dark-mode({
+            --_pa-item-fc: var(--black-400);
+            --_pa-item-bg-hover: var(--black-250);
+            --_pa-item-fc-hover: var(--black-500);
+        });
+
+        .highcontrast-mode({ 
+            text-decoration: none;
+            --_pa-item-fc: var(--black-600);
+            --_pa-item-bg-hover: var(--black-200);
+            --_pa-item-fc-hover: var(--black);
+        });
+
+        .highcontrast-dark-mode({
+            --_pa-item-fc: var(--black-400);
+            --_pa-item-bg-hover: var(--black-300);
+            --_pa-item-fc-hover: var(--black-500);
+        });
 
         // MODIFIERS
         &.is-selected {
-            --_pa-item-bg: var(--theme-primary);
-            --_pa-item-bc: transparent;
-            --_pa-item-fc: var(--white);
+            --_pa-item-bc: #201C1D;
+            --_pa-item-fc: #201C1D; // Update to grey-600 when colors are updated
+            --_pa-item-bg: transparent;
             --_pa-item-bg-focus: var(--theme-primary-400);
+            --_pa-item-bb: var(--su-static2);
+
+            .dark-mode({
+                --_pa-item-fc: var(--black-600);
+                --_pa-item-bc: var(--black-600);
+                --_pa-item-bg: var(--black-300);
+            });
+
+            .highcontrast-mode({
+                --_pa-item-fc: var(--black);
+                --_pa-item-bc: var(--black);
+                --_pa-item-bg: var(--black-400);
+            });
+
+            .highcontrast-dark-mode({
+                --_pa-item-fc: var(--black-500);
+                --_pa-item-bc: var(--black-500);
+                --_pa-item-bg: var(--black-300);
+            });
         }
         &.s-pagination--item__clear {
             --_pa-item-bg: transparent;
             --_pa-item-bc: transparent;
-            --_pa-item-fc: inherit;
+            --_pa-item-fc: #6B6D73;
+
+            .dark-mode({
+                --_pa-item-fc: var(--black-400);
+            });
+
+            .highcontrast-mode({
+                --_pa-item-fc: var(--black-600);
+            });
+
+            .highcontrast-dark-mode({
+                --_pa-item-fc: var(--black-400);
+            });
+        }
+        &.is-next {
+            background-color: #F0EFED;
+            border-radius: var(--br-circle);
```

Comment:
I'd set this as a psuedo private custom property with a value of `unset` at the top of the file and reset the value here to prevent cascade issues.